src/layouts/_proxied-dot-io/consul/index.tsx

```diff
@@ -40,67 +37,67 @@ function ConsulIoLayout({ children, data }: Props): React.ReactElement {
 			/>
 
 			<Min100Layout footer={<Footer openConsentManager={openConsentManager} />}>
-				<ProductMetaProvider product={productData.slug as Products}>
```

Comment:
Can we add back the `ProductMetaProvider`? It's necessary for certain components to render with the right product context.

Reply:
Whoops, good catch https://github.com/hashicorp/dev-portal/pull/669/commits/8eaf40fe490275150324b7c5211bfac3389381a1